projects/optic/src/commands/diff/diff.ts

```diff
@@ -198,6 +188,21 @@ const runDiff = async (
     },
   };
 
+  if (specResults.changes.length === 0) {
+    console.log('No changes were detected');
+  } else {
+    console.log('');
+  }
+
+  if (options.json) {
```

Comment:
If we're running this in JSON mode, should there be any other output? I presume we only want to dump a single json object out if `--json` is passed in?

Additionally, do you think it's valid to allow `--json` + `--web` opts to be specified at the same time?